### DISCUSSION TITLE
File acquisition failed

### DISCUSSION THREAD

**Discussion by dollarkillerx** [juicedata/juicefs#435] [General]
The current three servers are all under the same intranet, and all use enterprise-level SSDs
s1: os: ubuntu20 Deployment application: redis:6-alpine, juicefs version 0.13.0
s2: os: windows_server_2020 Deploy application: juicefs version 0.13.0 as a service provider node
s3: os: windows_server_2020 Deployment application: juicefs version 0.13.0

Experiment 1:
s2: run:
```
.\juicefs.exe format redis://192.168.31.35/1 data --bucket H:tmp
.\juicefs.exe mount redis://192.168.31.35/1 Z:
```
Then copy the 8G file to the Z mount disk

s1:
```
.\juicefs mount redis://192.168.31.35/1 /mut/z
```
Failed to copy files in the z directory to the local directory
The error message The system cannot find the path specified.
s3: Same as above, failed, the error message is the same

Experiment 2:
s2: run:
```
.\juicefs.exe format redis://192.168.31.35/2 data --bucket H:tmpv2 --block-size 1048576
.\juicefs.exe mount redis://192.168.31.35/2 G:
```
Then copy the 32G file to the G mount disk

s1:
```
.\juicefs mount redis://192.168.31.35/2 /mut/g
```
Failed to copy files in directory g to local directory
The error message The system cannot find the path specified.
s3: Same as above, failed, the error message is the same

ERROR: 
```
2021/05/22 08:06:12.353264 juicefs[9236] <WARNING>: fail to read chunkid 1 (off:0, size:1048576, clen: 67108864): get chunks/0/0/1_0_16777216: open H:tmp\data\chunks\0\0\1_0_16777216: The system cannot find the path specified.
2021/05/22 08:06:14.545882 juicefs[9236] <WARNING>: fail to read chunkid 1 (off:16777216, size:16777216, clen: 67108864): get chunks/0/0/1_1_16777216: open H:tmp\data\chunks\0\0\1_1_16777216: The system cannot find the path specified.
2021/05/22 08:06:14.545882 juicefs[9236] <ERROR>: read file 2: input/output error
```

**Comment by davies** [comment]
@dollarkillerx You are using local disk as the object store, which can only be accessed from current machine, so other machine can't read files. To use JuiceFS as a shared file system, you need to use a object store that can be accessed from multiple machines, for example, MinIO.
